tox.ini

```diff
@@ -32,6 +34,7 @@ deps =
     dj42: Django>=4.2,<4.3
     dj50: Django>=5.0,<5.1
     dj51: Django>=5.1,<5.2
+    dj52: Django>=5.2a1,<5.3
```

Comment:
Any particular reason to put `5.2a1` instead of just `5.2`? 🙂 If so, we should add a comment explaining why

Reply:
5.2a1 is because right now it's out in alpha. The 5.2 release comes out in April.

Reply:
Ah, I hadn't checked the release status recently 😅 Don't worry about it, then 👌

Reply:
```suggestion
    dj52: Django>=5.2a1,<5.3  # Use a1 to allow testing of the release candidates
```